src/main/java/com/crowdin/client/ai/AIApi.java

```diff
@@ -0,0 +1,150 @@
+package com.crowdin.client.ai;
+
+import com.crowdin.client.ai.model.FineTuningDatasetDownload;
+import com.crowdin.client.ai.model.FineTuningDatasetDownloadResponse;
+import com.crowdin.client.ai.model.FineTuningDatasetRequest;
+import com.crowdin.client.ai.model.FineTuningDatasetResponse;
+import com.crowdin.client.ai.model.FineTuningDatasetResponse.FineTuningDatasetData;
+import com.crowdin.client.ai.model.FineTuningEvent;
+import com.crowdin.client.ai.model.FineTuningEventResponseList;
+import com.crowdin.client.ai.model.FineTuningJob;
+import com.crowdin.client.ai.model.FineTuningJobRequest;
+import com.crowdin.client.ai.model.FineTuningJobResponseList;
+import com.crowdin.client.ai.model.FineTuningJobResponseObject;
+import com.crowdin.client.core.CrowdinApi;
+import com.crowdin.client.core.http.HttpRequestConfig;
+import com.crowdin.client.core.model.ClientConfig;
+import com.crowdin.client.core.model.Credentials;
+import com.crowdin.client.core.model.ResponseList;
+import com.crowdin.client.core.model.ResponseObject;
+
+public class AIApi extends CrowdinApi {
+
+    public AIApi(Credentials credentials) {
+        super(credentials);
+    }
+
+    public AIApi(Credentials credentials, ClientConfig clientConfig) {
+        super(credentials, clientConfig);
+    }
+
+    /**
+     * @param userId user identifier
+     * @param aiPromptId AI prompt identifier
+     * @param jobIdentifier AI prompt fine-tuning dataset generation identifier, consists of 36 characters
+     * @return dataset generation status
+     * @see <ul>
+     *     <li><a href="https://support.crowdin.com/developer/api/v2/#tag/AI/operation/api.users.ai.prompts.fine-tuning.datasets.get" target="_blank"><b>API Documentation</b></a></li>
+     *     <li><a href="https://support.crowdin.com/developer/enterprise/api/v2/#tag/AI/operation/api.ai.prompts.fine-tuning.datasets.get" target="_blank"><b>Enterprise API Documentation</b></li>
+     * </ul>
+     * */
+
+    public ResponseObject<FineTuningDatasetData> getFineTuningDatasetGenerationStatus(
+        final long userId, final long aiPromptId, final String jobIdentifier) {
+        String url = String.format("%s/users/%d/ai/prompts/%d/fine-tuning/datasets/%s", this.url, userId, aiPromptId, jobIdentifier);
```

Comment:
For Crowdin Enterprise, the `/users/{userId}` is missing in the request URL. Please take a look at the `ReportsApi.java` implementation, particularly the `getReportArchivesPath`.